.genx/templates/route.hbs

```diff
@@ -2,13 +2,38 @@ import { customElement, FASTElement } from '@microsoft/fast-element';
 import { {{pascalCase route.name}}Styles as styles } from './{{kebabCase route.name}}.styles';
 import { {{pascalCase route.name}}Template as template } from './{{kebabCase route.name}}.template';
 
+{{#if route.fdc3}}
+import { Events } from '@ag-grid-community/core';
+import { GRID_READY_EVENT, ZeroGridPro } from '@genesislcap/foundation-zero-grid-pro';
+{{/if}}
+
 @customElement({
   name: '{{kebabCase route.name}}-route',
   template,
   styles,
 })
 export class {{pascalCase route.name}} extends FASTElement {
+
   constructor() {
     super();
   }
+
+  {{#if route.fdc3}}
+  public tile1grid1!: ZeroGridPro;
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.tile1grid1.addEventListener(GRID_READY_EVENT, () => {
+      this.tile1grid1.gridApi.addEventListener(Events.EVENT_ROW_CLICKED, (e) =>
```

Comment:
we need to check this code block to check if we are in an fdc3 container

`if (window.fdc3)`

Reply:
yeah, I've just set up the foundation for this branch - you'll need to apply the actual changes needed on top